benchmark/feldera-sql/benchmarks/nexmark/table.sql

```diff
@@ -20,8 +20,8 @@ CREATE TABLE person (
                 "options": {{
                     "events": {events},
                     "threads": {cores},
-                    "batch_size": 10000,
-                    "synchronize_threads": true
+                    "batch_size": 16000,
```

Comment:
How sensitive is performance to this number? 16k (instead of 10k) feels oddly specific.

Reply:
16k was a "round number" for my testing because I was using 16 cores.

I could change it to `{cores}000` but I only tested with 16 cores on my particular hardware.

Reply:
The batch size is spread across the threads, so 16000 for 16 threads means 1000 per thread per batch.

Reply:
Should batch-size-per-worker be the parameter then?

Reply:
It can be. I pushed an update.